<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBizSettingPage</class>
 <widget class="QWidget" name="GBSBizSettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>840</y>
     <width>1141</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnConfirm">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>13</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread: pad, 
        x1: 0, y1: 0, x2: 1, y2: 1, 
        stop: 0.1054 #1B2846, 
        stop: 0.9062 #000000
    ); /* 渐变背景 */
    border: 1px solid white;       /* 边框为白色 */
    border-radius: 5px;            /* 圆角半径 */
    font-size: 16px;               /* 字体大小 */
    color: #FFFFFF;                /* 字体颜色 */
}

QPushButton:pressed {
    background-color: #D1D8DD;     /* 按下时的背景颜色 */
    padding-left: 3px;             /* 左侧内边距 */
    padding-top: 3px;              /* 顶部内边距 */
    background-repeat: no-repeat;  /* 禁止背景重复 */
    background-position: center;   /* 背景居中 */
}


</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>13</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread: pad, 
        x1: 0, y1: 0, x2: 1, y2: 1, 
        stop: 0.1054 #1B2846, 
        stop: 0.9062 #000000
    ); /* 渐变背景 */
    border: 1px solid white;       /* 边框为白色 */
    border-radius: 5px;            /* 圆角半径 */
    font-size: 16px;               /* 字体大小 */
    color: #FFFFFF;                /* 字体颜色 */
}

QPushButton:pressed {
    background-color: #D1D8DD;     /* 按下时的背景颜色 */
    padding-left: 3px;             /* 左侧内边距 */
    padding-top: 3px;              /* 顶部内边距 */
    background-repeat: no-repeat;  /* 禁止背景重复 */
    background-position: center;   /* 背景居中 */
}


</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>13</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread: pad, 
        x1: 0, y1: 0, x2: 1, y2: 1, 
        stop: 0.1054 #1B2846, 
        stop: 0.9062 #000000
    ); /* 渐变背景 */
    border: 1px solid white;       /* 边框为白色 */
    border-radius: 5px;            /* 圆角半径 */
    font-size: 16px;               /* 字体大小 */
    color: #FFFFFF;                /* 字体颜色 */
}

QPushButton:pressed {
    background-color: #D1D8DD;     /* 按下时的背景颜色 */
    padding-left: 3px;             /* 左侧内边距 */
    padding-top: 3px;              /* 顶部内边距 */
    background-repeat: no-repeat;  /* 禁止背景重复 */
    background-position: center;   /* 背景居中 */
}


</string>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1141</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>基础</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>直播输出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>音视频</string>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1141</width>
     <height>731</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources>
  <include location="../../forms/obs.qrc"/>
 </resources>
 <connections/>
</ui>
